cli/src/pcluster/utils.py

```diff
@@ -549,6 +549,17 @@ def get_supported_az_for_multi_instance_types(instance_types):
     return result
 
 
+def get_common_supported_az_for_multi_instance_types(instance_types):
```

Comment:
Because we are handle multi insatnce types. We need the function to find the common availability zone to create subnets. I guess you prefer putting them in boto3 layer?

Reply:
We can keep it here for now, thanks